contracts/core/liquidators/TWAPLiquidator.sol

```diff
@@ -49,36 +49,52 @@ contract TWAPLiquidator is
 {
     using SafeMath for uint256;
 
+    /* ============ Events ============ */
+    event ChunkAuctionIterated(
+        address indexed rebalancingSetToken,
+        uint256 orderRemaining,
+        uint256 startingCurrentSets
+    );
+
+    event ChunkSizeBoundUpdated(
+        address assetOne,
+        address assetTwo,
+        uint256 lowerBound,
+        uint256 upperBound
+    );
+
     ICore public core;
     string public name;
+    // Maps RebalancingSetToken to it's auction state
     mapping(address => TWAPAuction.TWAPState) public auctions;
 
     /* ============ Modifier ============ */
-    modifier isValidSet() {
-        requireValidSet(msg.sender);
+    modifier onlyValidSet() {
+        require(
```

Comment:
FYI - The reason I previously moved it into its own function is that each require and its text duplicates increases the bytecode size. If it deploys and compiles for now, this is fine